src/main/java/net/minecraftforge/common/config/ConfigManager.java

```diff
@@ -117,7 +118,7 @@ public static void loadData(ASMDataTable data)
         }
     }
 
-    public static void load(String modid, Config.Type type)
+    public static void sync(String modid, Config.Type type)
     {
```

Comment:
The fundemental concept of changing this load to a sync that is called multiple times is bad because we're recreating the config object everytime. Which destroies the defaults and anything like that.

We need a load, which is called once by our config management system which loads the comments, loads the defaults and ranges, then we need one where it just syncs it between Class->Config for when modder change the class directly.
And then we need one that syncs Config->Class for when the GUI edits things.